.github/workflows/gpu.yml

```diff
@@ -20,20 +19,33 @@ jobs:
         build_type: [Release]
         compiler: [{c: gcc, cxx: g++}]
         shared_library: ['ON', 'OFF']
-    # TODO enable testing on DSS-WINDOWS
-    runs-on:  [ "DSS-LEVEL_ZERO", "DSS-UBUNTU" ]
+        include:
+          - os-labels: ["DSS-LEVEL_ZERO", "DSS-UBUNTU"]
+            os: 'UBUNTU'
+            compiler: [{c: gcc, cxx: g++}]
+            cmake_toolchain_file: 'OFF'
```

Comment:
not sure if you need this toolchain here, not sure if it will work with value "OFF"

You can omit this param totally for Ubuntu and just set, e.g., `extra_arg: -DCMAKE_TOOLCHAIN_FILE=<path>` just for Windows (extra arg will be then used at the end of the CMake command)